web-bluetooth/link-loss.js

```diff
@@ -0,0 +1,70 @@
+var alertLevelCharacteristic;
+
+function onReadButtonClick() {
+  log('Requesting Bluetooth Device...');
+  navigator.bluetooth.requestDevice(
+    {filters: anyDevice(), optionalServices: ['link_loss']})
```

Comment:
Why not filter to devices that support the 'link_loss' service? That's a pretty rare service, and I don't see a reason devices would be likely to support it without advertising it.